pvlib/iotools/psm3.py

```diff
@@ -296,12 +331,25 @@ def parse_psm3(fbuf):
     tz = 'Etc/GMT%+d' % -metadata['Time Zone']
     data.index = pd.DatetimeIndex(dtidx).tz_localize(tz)
 
+    if map_variables is None:
+        warnings.warn(
+            'PSM3 variable names will be renamed to pvlib conventions by '
+            'default starting in pvlib 0.11.0. Specify map_variables=True '
+            'to enable that behavior now, or specify map_variables=False '
+            'to hide this warning.', pvlibDeprecationWarning)
+        map_variables = False
+    if map_variables:
+        data = data.rename(columns=VARIABLE_MAP)
+        metadata['latitude'] = metadata.pop('Latitude')
+        metadata['longitude'] = metadata.pop('Longitude')
+        metadata['elevation'] = metadata.pop('Elevation')
```

Comment:
Interesting... I'll make a follow-up issue requesting that this term be added to the [Variables and Symbols list](https://pvlib-python.readthedocs.io/en/latest/user_guide/variables_style_rules.html#variables-style-rules).

As it is right now, it would be hard for users to figure out that `map_variables` return altitude instead of elevation. Is there a good way to denote that? E.g., adding it to the ``VARIABLE_MAP``?